JS/Lesson35/onur/my-app/src/components/NewRecipe/styles.css

```diff
@@ -1,10 +1,50 @@
 .new-recipe-form {
   margin: 20px auto;
-  max-width: 680px;
+  max-width: 600px;
+  padding: 1rem;
+  border-radius: 10px;
+  background-color: #acb4af;
```

Comment:
For the next projects, try declaring CSS variables in index.css file and then use colors as variables here. It should be easier that adding color codes one by one. Also, it would be a more scalable approach :)